Jellyfin.Server/Migrations/Routines/RemoveBuggedExtras.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.IO;
+
+using MediaBrowser.Controller;
+using Microsoft.Extensions.Logging;
+using SQLitePCL.pretty;
+
+namespace Jellyfin.Server.Migrations.Routines
+{
+    /// <summary>
+    /// Remove duplicate entries which were caused by a bug where a file was considered to be an "Extra" to itself.
+    /// </summary>
+    internal class RemoveBuggedExtras : IMigrationRoutine
+    {
+        private const string DbFilename = "library.db";
```

Comment:
This constant should probably be defined in `SqliteItemRepository`

Reply:
It is defined there and I tried to use it first, but to extract it I would have to refactor it a bit and get access to that class here, which sounds a bit overcomplicating IMHO.

Reply:
Okay, that's fine 👍